2022-06-01 13:55:55,548:INFO:foobar:Test to standard out
2022-06-01 13:55:55,548:ERROR:foobar:Traceback (most recent call last):
2022-06-01 13:55:55,548:ERROR:foobar:  File "/Users/joelyancey/glanceem_swift/swift-ir/source/PySide6/run.py", line 73, in <module>
2022-06-01 13:55:55,549:ERROR:foobar:raise Exception('Test to standard error')
2022-06-01 13:55:55,549:ERROR:foobar:Exception
2022-06-01 13:55:55,549:ERROR:foobar::
2022-06-01 13:55:55,549:ERROR:foobar:Test to standard error
2022-06-01 13:55:59,697:INFO:foobar:Test to standard out
2022-06-01 13:55:59,697:ERROR:foobar:Traceback (most recent call last):
2022-06-01 13:55:59,697:ERROR:foobar:  File "/Users/joelyancey/glanceem_swift/swift-ir/source/PySide6/run.py", line 73, in <module>
2022-06-01 13:55:59,697:ERROR:foobar:raise Exception('Test to standard error')
2022-06-01 13:55:59,703:ERROR:foobar:Exception
2022-06-01 13:55:59,703:ERROR:foobar::
2022-06-01 13:55:59,703:ERROR:foobar:Test to standard error
2022-06-01 13:57:03,860:INFO:foobar:Test to standard out
2022-06-01 13:57:03,860:ERROR:foobar:Traceback (most recent call last):
2022-06-01 13:57:03,861:ERROR:foobar:  File "/Users/joelyancey/glanceem_swift/swift-ir/source/PySide6/run.py", line 78, in <module>
2022-06-01 13:57:03,861:ERROR:foobar:main_win = interface.MainWindow(title="GlanceEM_SWiFT") # no more control_model
2022-06-01 13:57:03,861:ERROR:foobar:  File "/Users/joelyancey/glanceem_swift/swift-ir/source/PySide6/interface.py", line 3936, in __init__
2022-06-01 13:57:03,861:ERROR:foobar:raise Exception('Test to standard error')
2022-06-01 13:57:03,861:ERROR:foobar:Exception
2022-06-01 13:57:03,862:ERROR:foobar::
2022-06-01 13:57:03,862:ERROR:foobar:Test to standard error
2022-06-01 13:57:56,287:INFO:foobar:Test to standard out
2022-06-01 13:57:56,287:ERROR:foobar:Traceback (most recent call last):
2022-06-01 13:57:56,287:ERROR:foobar:  File "/Users/joelyancey/glanceem_swift/swift-ir/source/PySide6/run.py", line 78, in <module>
2022-06-01 13:57:56,287:ERROR:foobar:main_win = interface.MainWindow(title="GlanceEM_SWiFT") # no more control_model
2022-06-01 13:57:56,287:ERROR:foobar:  File "/Users/joelyancey/glanceem_swift/swift-ir/source/PySide6/interface.py", line 3936, in __init__
2022-06-01 13:57:56,288:ERROR:foobar:raise Exception('Test to standard error')
2022-06-01 13:57:56,288:ERROR:foobar:Exception
2022-06-01 13:57:56,288:ERROR:foobar::
2022-06-01 13:57:56,288:ERROR:foobar:Test to standard error
2022-06-01 13:58:48,449:INFO:root:Test to standard out
2022-06-01 13:58:48,449:ERROR:root:Traceback (most recent call last):
2022-06-01 13:58:48,449:ERROR:root:  File "/Users/joelyancey/glanceem_swift/swift-ir/source/PySide6/run.py", line 78, in <module>
2022-06-01 13:58:48,450:ERROR:root:main_win = interface.MainWindow(title="GlanceEM_SWiFT") # no more control_model
2022-06-01 13:58:48,450:ERROR:root:  File "/Users/joelyancey/glanceem_swift/swift-ir/source/PySide6/interface.py", line 3936, in __init__
2022-06-01 13:58:48,451:ERROR:root:raise Exception('Test to standard error')
2022-06-01 13:58:48,451:ERROR:root:Exception
2022-06-01 13:58:48,451:ERROR:root::
2022-06-01 13:58:48,451:ERROR:root:Test to standard error
